monitor/urls.py

```diff
@@ -18,17 +18,16 @@
 #     See AUTHORS file.
```

Comment:
General comment: please run pylint over your code... It will help you follow PEP8 and apply a bunch more useful rules...